src/data/faq.json

```diff
@@ -1052,35 +1054,42 @@
                                 ]
                             },
                             {
-                                "title": "Can the Corona-Warn-App read and process the QR codes from luca?",
+                                "title": "The luca app does not currently offer event registration. Can the Corona-Warn-App still read and process the QR codes from luca?",
                                 "anchor": "check_in_luca_codes",
                                 "active": false,
                                 "textblock": [
-                                    "Yes. The Corona-Warn-App can scan QR codes issued for event registration using the luca app and perform its own check-in process from the CWA."
+                                    "<b>Updated 15 July, 2022</b>",
+                                    "Since spring 2022, the event registration (check-in functionality) of the luca app is no longer possible for the time being. However, the Corona-Warn-App can scan QR codes issued for event registration using the luca app and perform its own check-in process from the CWA."
                                 ]
                             },
                             {
                                 "title": "Can luca also read and process QR codes from the Corona-Warn-App?",
                                 "anchor": "check_in_luca_cwa_codes",
                                 "active": false,
                                 "textblock": [
-                                    "No, this is not possible. The luca system transmits the contact data required for this purpose from visitors to the event or establishment in question to the health authorities as part of contact tracking, which must be health authority-compliant. However, the CWA does not provide for the transmission of contact data. Thus, QR codes are needed for this process, which are different from those required by the CWA."
+                                    "<b>Updated 15 July, 2022</b>",
+                                    "Note: Since spring 2022, the event registration (check-in functionality) of the luca app is no longer possible for the time being. Even when this functionality was still available in the luca app, it was not possible for the luca app to read and process QR codes from the CWA.<hr>",

```

Comment:
@Ein-Tim 
Yes, it is strange. I commented about it separately in https://github.com/corona-warn-app/cwa-website/pull/2757#issuecomment-1185663618, since it is used multiple times.

The German text isn't so good either, so if this gets changed then the English text could also be completely adapted.

Reply:
First sentence has been changed to "The luca app no longer offers event registration (check-in functionality) as of spring 2022.", as suggested by @floscher in another conversation, in all affected FAQs in https://github.com/corona-warn-app/cwa-website/pull/2757/commits/0b254036e8d8d1fad6ce38c9d1614b4b9313aac2, thanks!